spec/govuk_docker_cli_spec.rb

```diff
@@ -97,10 +97,8 @@
 
     context "without a variation argument" do
       it "runs in the app stack" do
-        expect(Commands::Run)
-          .to receive(:new).with(stack: "app", verbose: false)
-          .and_return(command_double)
-        expect(command_double).to receive(:call).with(no_args)
+        expect(Commands::Startup).to receive(:new).and_return(command_double)
```

Comment:
Where these changes meant to be in commit 878283ed53bf5f52b552e0500d3e94aefe932c97?

Reply:
Yep... I've fixed that now.